JEduardoRJx/palette-picker-backend
# Backend setup

What’s this PR do?
#### Where should the reviewer start?
Look at the package.json and you can see the dependencies that have been installed. You can also observe the app.js, server,js, knexfile.js, and .babelrc.
#### How should this be manually tested?
Pull this branch down locally and run npm start. You should see a message from Tone Zone on localhost:3000
#### Any background context you want to provide?
Used this epic summary to setup backend resource @JEduardoRJx created using the lessons from class: https://gist.github.com/JEduardoRJx/2de8d199b3836958ee90bf822da18e20
#### What are the relevant tickets?
#14 
#### Screenshots (if appropriate)
<img width="288" alt="Screen Shot 2019-12-04 at 3 00 04 PM" src="https://user-images.githubusercontent.com/47537744/70185345-d27cd500-16a6-11ea-8201-5b225d3d9cfd.png">

#### Questions:
NA

####Notes:
The next step will be setting up database on our local machines and seeding them.
